### PR TITLE
special migration -> async migration

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

"Special migration" was not the best name. We're now calling these async migrations. Background migrations is another option if anyone is strongly opposed to async migrations (that's what GitLab calls them).

## How did you test this code?

Relying on existing tests

### CODE REVIEW

**Review by Twixes — COMMENTED** [review]
Found quite a bit of `SPECIAL_MIGRATION` occurrences left, is it feasible to rename those too?

**Comment by yakkomajuri** [comment]
yup! @Twixes

**Comment by yakkomajuri** [comment]
✅ anyone? @tiina303 @Twixes

**Review by tiina303 — APPROVED** [review]
LGTM

File: posthog/migrations/0191_rename_specialmigration_asyncmigration.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0190_experiment"),
+    ]
+
+    operations = [
+        migrations.RenameModel(old_name="SpecialMigration", new_name="AsyncMigration",),
```
[the remark this answers is not in the document]
Comment (yakkomajuri) [reply posthog/migrations/0191_rename_specialmigration_asyncmigration.py:13]: This is safe and we've done it before on 0178.

Here are the scenarios:

1. Fresh install: We'll create the model with the old name and immediately rename it during the migrate step
2. Upgrade from having SpecialMigration named model: rename the table

There isn't a case where `posthog_specialmigration` doesn't exist and we try to rename it. The only way for that to happen is if someone manually meddled with the DB

**Pull request merged**